### ISSUE TITLE
Dropping connection

### ISSUE THREAD

**Issue by ABredhauer** [klaasnicolaas/home-assistant-glow#225]
I've had my glow up and running for about a week now and have noticed it will often stop reporting data showing as offline through ESP home and unavailable via Home Assistant. I setup a ping every 30s and can see that even when it is showing as offline it still responds to pings and I've never dropped a packet. 

Any advice on how to help my diagnose this?

**Comment by klaasnicolaas** [comment]
Is there a lot of distance between your glow and your router?

**Comment by Kimble8650** [comment]
I have the same issue. I've had my glow up and running since January without any bigger hiccups. Now I've noticed that the connection is much more unreliable for maybe a month or so. The only thing that has changed in my setup is the home assistant version.

I have not updated the glow firmware, have not moved it, nor have I moved any wifi access points. Weird.

**Comment by klaasnicolaas** [comment]
Strange, I've been running it for years now and never had any problems with the connection. Would therefore not immediately know what this could be, how are the RSSI values?

**Comment by Kimble8650** [comment]
Mostly around -67dBm to -76dBm with an occasional outlier. But as said, wifi coverage has never been an issue before. I too am clueless what could be the culprit. I just started investigating this today, so I'm not very far along in the process :D

**Comment by Kimble8650** [comment]
Ok, so I have made some progress on my issue
I noticed that my wifi was not as responsive as usual on my other devices, so I started looking at my Unify Controller, and low and behold, suddenly all my APs had been dropped from the controllers sphere of influence.
So I readopted the APs and my wifi experience went up significantly. This did not fix my issue entirely, but it did make a huge difference. See picture below and try to find where I readopted my APs =).

<img width="1449" alt="Screenshot 2022-11-14 at 8 57 04" src="https://user-images.githubusercontent.com/9073143/201594915-4fa42627-11ef-4d08-adfe-11ae0caaabf5.png">

So for me this has been at least in part, a wifi routing issue. I guess the APs started routing independently on the same SSIDs, which caused a lot of conflicts.

The quest goes on.
